apps/platform_runner/scripts/local_postgres_setup/create_users_and_databases.sql

```diff
@@ -0,0 +1,25 @@
+--- This script is for setting up users and databases for local testing of the platform --
```

Comment:
can this move into a `test/support` directory to clearly indicate it's purpose?

Reply:
Can do!